thehive/app/org/thp/thehive/controllers/v0/ObservableCtrl.scala

```diff
@@ -417,7 +417,7 @@ class PublicObservable(
     )
     .property("message", UMapping.string)(_.field.updatable)
     .property("tlp", UMapping.int)(_.field.updatable)
-    .property("dataType", UMapping.string)(_.field.readonly)
+    .property("dataType", UMapping.string)(_.field.updatable)
```

Comment:
Instead of `updatable`, you can user the method `custom` which give you the full control of the update process.